### PR TITLE
Add component configuration for My Wallet

### PR DESCRIPTION
**Pull request by maassenbas**
## Tested scenarios

**Fixed issue**:

### CODE REVIEW

**Review by wboereboom — APPROVED** [review]

**Comment by wboereboom** [comment]
I noticed some file names are camel cased, whilst others are snake cased. Is this due to some Apex standards?

**Comment by maassenbas** [comment]
Yes, this is a naming convention in the CloudCraze platform.

File: force-app/main/default/classes/cc_adyencc_hk_Payment.cls [hunk 1]
```diff
@@ -22,16 +22,20 @@ global class cc_adyencc_hk_Payment extends ccrz.cc_hk_Payment {
...
-        //Remove suffix and convert to string
-        String jsonPaymentMethodsResponse = JSON.serialize(paymentMethodsResponse, true);
-        String stringPaymentMethodsResponse = Util.makeAdyenCompatible(jsonPaymentMethodsResponse);
         Map<String,Object> returnData =  super.fetchPaymentTypes(inputData);
-        Map<String,Object> paymentTypes = (Map<String,Object>)returnData.get('paymentTypes');
-        Map<String,Object> adyencc = (Map<String,Object>)paymentTypes.get('adyencc');
-        adyencc.put('adyenPaymentMethods', stringPaymentMethodsResponse);
-        adyencc.put('adyenClientKey', AdyenUtil.getClientKey());
-        adyencc.put('adyenEnvironment', AdyenUtil.getAdyenEnvironment());
+        if(inputData.get('paymentTypesContext') == 'co'){
```
Comment (wboereboom) [inline force-app/main/default/classes/cc_adyencc_hk_Payment.cls:26]: Are there more of these constants? Maybe we could consider creating some kind of Enum class. 'co' is difficult to link to any kind of context

Comment (maassenbas) [reply force-app/main/default/classes/cc_adyencc_hk_Payment.cls:27]: Added a code comment there

File: force-app/main/default/classes/cc_adyencc_hk_Payment.cls [hunk 2]
```diff
@@ -22,16 +22,21 @@ global class cc_adyencc_hk_Payment extends ccrz.cc_hk_Payment {
...
-        //Remove suffix and convert to string
-        String jsonPaymentMethodsResponse = JSON.serialize(paymentMethodsResponse, true);
-        String stringPaymentMethodsResponse = Util.makeAdyenCompatible(jsonPaymentMethodsResponse);
         Map<String,Object> returnData =  super.fetchPaymentTypes(inputData);
-        Map<String,Object> paymentTypes = (Map<String,Object>)returnData.get('paymentTypes');
-        Map<String,Object> adyencc = (Map<String,Object>)paymentTypes.get('adyencc');
-        adyencc.put('adyenPaymentMethods', stringPaymentMethodsResponse);
-        adyencc.put('adyenClientKey', AdyenUtil.getClientKey());
-        adyencc.put('adyenEnvironment', AdyenUtil.getAdyenEnvironment());
+        //paymentTypesContext co = checkout and paymentTypesContext wlt = wallet
```
Comment (zaiddreakh) [inline force-app/main/default/classes/cc_adyencc_hk_Payment.cls:26]: Was this comment left on purpose?

**Review by zaiddreakh — APPROVED** [review]

File: force-app/main/default/classes/cc_adyencc_hk_Payment.cls (hunk 2 above)
Comment (zaiddreakh) [reply force-app/main/default/classes/cc_adyencc_hk_Payment.cls:26]: oh I just noticed Wouter's review :P

**Pull request merged**